tests/integration/transformers/pii/test_faker.py

```diff
@@ -0,0 +1,105 @@
+
+import numpy as np
+import pandas as pd
+
+from rdt.transformers.pii import PIIAnonymizer
+
+
+def test_piifaker():
```

Comment:
should be `piianonymizer`. Can we do one more pass through to make sure all of the names are correct?